sendgrid/helpers/mail/tracking_settings.py

```diff
@@ -1,5 +1,6 @@
 class TrackingSettings(object):
     """Settings to track how recipients interact with your email."""
+
```

Comment:
Accidental change in merge commit - added a newline between class docstring and `__init__` in several places, but this is consistent with the rest of the code (and probably some PEP rule).